easybuild/easyblocks/c/cp2k.py

```diff
@@ -392,7 +392,7 @@ def configure_common(self):
 
                 # build libint wrapper
                 cmd = "%s -c libint_cpp_wrapper.cpp -I%s/include" % (libintcompiler, libint)
-                if not run_cmd(cmd, log_all=True, simple=True):
+                if not run_shell_cmd(cmd):
```

Comment:
@branfosj since we don't have `simple=True` any more, would `run_shell_cmd(cmd)` ever be falsy? I imagine we need to check the exit code?

Reply:
```suggestion
                res = run_shell_cmd(cmd)
                if res.exit_code:
```

Reply:
Done in bd5fcd4

Reply:
and ddaa3a2 to remove the `not`

Reply:
@branfosj hmm, do we also need `fail_on_error=False`?

Reply:
Yes and in another spot in this easyconfig. :(

Reply:
e3c06ef

I'll set a new test report going...

Reply:
thanks :)